src/main/kotlin/org/elm/ide/toolwindow/ElmWorkspacePanel.kt

```diff
@@ -65,7 +65,7 @@ class ElmWorkspacePanel(private val project: Project) : SimpleToolWindowPanel(tr
 
 
     init {
-        setToolbar(createToolbar())
+        toolbar = createToolbar()
```

Comment:
Knowing nothing about Kotlin, this change looks a bit suspicious. Previously, it looks like a side effect happened in `setToolbar`. Now it looks like the side effect does not happen, and we assign to an unused (?) variable called toolbar?

Reply:
Ah yeah, my bad. (Well, I'll take responsibility for it anyway). The IDE was prompting me to change it (for some reason) so I just followed it. I've reverted the change.